### PR TITLE
[GNA] Added error trapping

### PR DESCRIPTION
**Pull request by admitriev-gna**
Now, if colomns_in receives an input more than 8, then it will throw an error of an unsupported parameter.

### CODE REVIEW

**Comment by pavel-rodionov** [comment]
Could you please re-run the CI

**Comment by dorloff** [comment]
Please rebase again.

**Comment by dorloff** [comment]
No sense to rebase - it's failing on GNA tests. @admitriev-gna, please take a look at error messages.

File: inference-engine/src/gna_plugin/gna_graph_compiler.cpp [hunk 1]
```diff
@@ -351,6 +351,10 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
 
+    if (num_columns_in > 8) {
+        THROW_GNA_EXCEPTION << "GNA colomns_in should be less than 8 and now colomns_in = " << num_columns_in;
```
Comment (dorloff) [inline inference-engine/src/gna_plugin/gna_graph_compiler.cpp:355]: Several issues in the error message:
1. Should be "columns_in", not "colomns_in".
2. No need to mention GNA - it's already mentioned by the macro.
3. It claims that "it should be less than 8", but it's wrong. It should be not greater than 8, not less.
4. It's two low level. Why not explain what's wrong from the user's perspective?

Can we add a test case for this error message?
And please rebase.

File: inference-engine/src/gna_plugin/gna_graph_compiler.cpp [hunk 2]
```diff
@@ -828,6 +832,7 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
 
+
```
Comment (dorloff) [inline inference-engine/src/gna_plugin/gna_graph_compiler.cpp:835]: Why new empty line?

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/exec_layers.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    };
+
+    INSTANTIATE_TEST_CASE_P(smoke_BehaviorTests, ExecLayerTests,
+            ::testing::Combine(
+            ::testing::ValuesIn(netPrecisions),
+            ::testing::Values(CommonTestUtils::DEVICE_GNA),
+            ::testing::ValuesIn(configs)),
+            ExecLayerTests::getTestCaseName);
+
+}  // namespace
```
Comment (dorloff) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/exec_layers.cpp:24]: Add newline

File: inference-engine/tests/functional/plugin/shared/include/behavior/exec_layers.hpp [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        // Create ngraph function
+        auto params = ngraph::builder::makeParams(ngraph::element::f32, {{9, 256}});
+        auto const_mult = ngraph::builder::makeConstant(ngraph::element::f32, {}, {-1.0f});
+        auto mult = std::make_shared<ngraph::opset1::Multiply>(params[0], const_mult);
+        auto func = std::make_shared<ngraph::Function>(mult, params);
+        auto ie = InferenceEngine::Core();
+        InferenceEngine::CNNNetwork cnnNet(func);
+        ASSERT_THROW(ie.LoadNetwork(cnnNet, targetDevice), InferenceEngine::details::InferenceEngineException);
+    }
+} // namespace BehaviorTestsDefinitions
```
Comment (dorloff) [inline inference-engine/tests/functional/plugin/shared/include/behavior/exec_layers.hpp:23]: Add newline

File: inference-engine/src/gna_plugin/gna_graph_compiler.cpp (hunk 1 above)
Comment (dorloff) [reply inference-engine/src/gna_plugin/gna_graph_compiler.cpp]: What about note 4?

**Review by dorloff — APPROVED** [review]

**Comment by dorloff** [comment]
@esmirno, will you be reviewing this?

**Pull request closed without merging**